src/components/KBShortcut/KBShortcut.tsx

```diff
@@ -49,7 +49,7 @@ export class KBShortcut extends React.Component<KBProps> {
     shouldComponentUpdate() {
         return false;
     }
-    UNSAFE_componentWillReceiveProps(next_props) {
+    componentDidUpdate(next_props) {
```

Comment:
A difference between `componentWillReceiveProps` and `componentDidUpdate` is that the first is passed `nextProps` and the second is passed `prevProps, prevState`, so it seems to me like maybe the rest of this function needs to switch to using `this.props` instead of the now mislabeled `next_props` (since it's really `prevProps`?) I think this might cause subtle bugs as it is if the component is reused (perhaps such as when changing between games or something of the sort, I'm not sure though on a place where it'd reliably be reused for testing)

Reply:
Ah okay. Does the updated version look right?

Reply:
I think it does, but I'll want to bang on that a little before a merge